vignettes/trial_definition.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "Trial Definition"
+bibliography: vignettes.bib
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Trial Definition}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
```

Comment:
if `tidyverse` is needed (and maybe we actually only need a few package(s) from it) then just load here with `library()`

Reply:
I have removed all references to the metapackage `tidyverse`.  Instead, vignettes now
- Reference individual packages (eg `dplyr`) directly
- Use `requireNamespace()` to check that the required packages are installed and, if not, fail gracefully.